OIPA/api/activity/tests/test_save_serializers.py

```diff
@@ -2134,12 +2134,12 @@ def test_create_transaction(self):
                 "code": transaction_type.code,
                 "name": 'irrelevant',
             },
-            "recipient_countries": [{
+            "transaction_recipient_country": {
```

Comment:
Please use "recipient_country" as the key instead and remove the O2M "recipient_countries" field